addons/main/functions/fnc_drawDownedUnitIndicator.sqf

```diff
@@ -14,4 +14,13 @@ private _color = [
     if (_distance > GVAR(showDownedUnitIndicatorRange)) then {continue};
     private _sizeFinal = linearConversion [0, GVAR(showDownedUnitIndicatorRange), _distance, _size, _size/10, false];
     drawIcon3D [["\a3\ui_f\data\igui\cfg\actions\bandage_ca.paa", "\A3\ui_f\data\Map\VehicleIcons\pictureHeal_ca.paa"] select (_x getVariable [QGVAR(beingRevived), false]), _color, (ASLtoAGL visiblePositionASL _x) vectorAdd [0, 0, 0.5], _sizeFinal, _sizeFinal, 0, "", 0, 0, "RobotoCondensed", "center", true, 0, 0.5];
+    if (!isNull player) then {
```

Comment:
```suggestion
    if !(isNull player) then {
```

Reply:
reason: code style

Reply:
But then again, why even check if player is null here? The player avatar is not needed here as it takes the camera position for distance. if `getUnitTrait` is used on a null object it will *should* return the negative case.

Reply:
I'm just used to checking for NPEs, haha. If you think it's fine to remove then I'll remove it.